standard/standard_8.php

```diff
@@ -244,7 +244,6 @@ function ob_end_clean(): bool {}
  * @link https://php.net/manual/en/function.ob-get-flush.php
  * @return string|false the output buffer or false if no buffering is active.
  */
-#[Pure(true)]
```

Comment:
I suppose this is incorrect as well. The function relies on the current output buffering stack. If no output buffer is active, ob_get_flush() will return false. The output buffering state is part of PHP's global state.. Please correct me if I'm wrong.

Reply:
I see. Thank you! I agree.